Sources/hostmgr-helper/ManagedVirtualMachine.swift

```diff
@@ -0,0 +1,75 @@
+import Foundation
+import Virtualization
+import OSLog
+import Network
+import libhostmgr
+
+@MainActor
+class ManagedVirtualMachine {
```

Comment:
The idea here was to isolate the identity of a managed machine and handle its lifecycle - starting, configuring, tearing down, and cleaning up.